mqtt/mqtt-bridge/src/pump/egress.rs

```diff
@@ -14,6 +19,11 @@ pub use crate::client::MockPublishHandle as PublishHandle;
 #[cfg(not(test))]
 use crate::client::PublishHandle;
 
+use mqtt3::proto::Publication;
+use std::iter;
+
+const MAX_IN_FLIGHT: usize = 16;
```

Comment:
Check with @ancaantochi if that's should come from configuration

Reply:
She said you said it shouldn't be configurable. Do we expect customers will want to configure this? Do they currently do so for edgehub? If so, I think we can add it now.

Reply:
not for edgehub right. for another remote broker can potentially be different limits.

Reply:
I was just saying that if it is something commonly configured without the broker/bridge, we should add the configuration now.